src/backend/templates/tracks.hbs

```diff
@@ -837,6 +837,19 @@
       function loadVideoAndSlides(div, videoURL, slideURL){
         if(videoURL !== null && videoURL !== '') {
           let isVideoDisplayed = $('#desc-' + div).hasClass('in');
+
+          if (videoURL.indexOf('v=') !== -1 && videoURL.indexOf('&feature=youtu.be') === -1) {
+            videoURL = videoURL.split('v=')[1];
+          }
+          else if (videoURL.indexOf('v=') !== -1 && videoURL.indexOf('&feature=youtu.be') !== -1) {
+            const endIndex = videoURL.indexOf('&feature=youtu.be');
+            const startIndex = videoURL.indexOf('v=');
+            videoURL = videoURL.substring(startIndex+2, endIndex);
+          }
+          else if (videoURL.indexOf('https://youtu.be/') !== -1) {
+            videoURL = videoURL.split('https://youtu.be/')[1];
+          }
```

Comment:
Could you put this code in a common library script, so that it won't be [repeated 3 times](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming))?

It's often easier to do that when writing the code than it is to do it [later](https://github.com/fossasia/open-event-webapp/issues/1881#issuecomment-391686292). (In the future, we may not be so sure that the code is an exact match.)